.gitattributes

```diff
@@ -1,3 +1,91 @@
+# Source code
```

Comment:
are these changes related to pgvector support?

Reply:
I was having issues with Windows line endings corrupting bash scripts in the repository. I generally try to set all my line endings to LF to prevent having to run dos2unix on affected files.